api-testing/multicloud-api/src/main/java/com/soda/utils/signature/SodaV4Signer.java

```diff
@@ -0,0 +1,56 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils.signature;
+
+import uk.co.lucasweb.aws.v4.signer.Header;
+import uk.co.lucasweb.aws.v4.signer.HttpRequest;
+import uk.co.lucasweb.aws.v4.signer.Signer;
+import uk.co.lucasweb.aws.v4.signer.credentials.AwsCredentials;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class SodaV4Signer {
+
+    /**
+     * Generate v4 Signature.
+     *
+     * @param method     Method name e.g GET
+     * @param url        URL e.g https://www.opensds.io/
```

Comment:
Please update URL

Reply:
Can you please give me a e.g URL?

Reply:
I have used this url https://sodafoundation.io . Please review.